crates/benda/src/parser/builtins.rs

```diff
@@ -0,0 +1,3 @@
+pub enum BuiltIn {
```

Comment:
What is this supposed to be?

Reply:
I was trying to figure out a syntax for a switch with binding, but then I went through with another idea. Dead code for now.